### ISSUE TITLE
Playbook is not running - ERROR! Cannot reset connection

### ISSUE THREAD

**Issue by Vad1mo** [atosatto/ansible-dockerswarm#39]
I have difficulties applying the latest playbook from latest master. I always get a `ERROR! Cannot reset connection` error message. I am not able to figure out what exactly is wrong with it.

First there are this warning messages, which are not directly related to the problem I think.

```
TASK [ansible-dockerswarm : Kill the ansible_user active SSH connections] ********************************************************
 [WARNING]: when statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: ansible_user is
defined and ansible_user in docker_admin_users and ansible_user in "{{ addtogroup.results | selectattr('changed') |
map(attribute='item') | list }}" and ansible_version.major <= 2 and ansible_version.minor <3

 [WARNING]: when statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: ansible_user is
defined and ansible_user in docker_admin_users and ansible_user in "{{ addtogroup.results | selectattr('changed') |
map(attribute='item') | list }}" and ansible_version.major <= 2 and ansible_version.minor <3

skipping: [cluser-node-1]
skipping: [cluser-node-2]
 [WARNING]: when statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: ansible_user is
defined and ansible_user in docker_admin_users and ansible_user in "{{ addtogroup.results | selectattr('changed') |
map(attribute='item') | list }}" and ansible_version.major <= 2 and ansible_version.minor <3

skipping: [cluser-node-3]
ERROR! Cannot reset connection:
Control socket connect(/Users/v/.ansible/cp/none): No such file or directory
```

**Comment by atosatto** [comment]
Is this still the case with the latest master release?

**Comment by Vad1mo** [comment]
Yes its on that latest master, I force installed the latest master this morning . 

```
ansible-galaxy install --force git+https://github.com/atosatto/ansible-dockerswarm.git,master
- changing role ansible-dockerswarm from None to master
- extracting ansible-dockerswarm to /Users/v/.ansible/roles/ansible-dockerswarm
- ansible-dockerswarm (master) was installed successfully
➜  clusteros git:(master) ✗ ansible-galaxy list
- ansible-dockerswarm, master
```

**Comment by Vad1mo** [comment]
I dug bit into the topic and it is related to the PR #34

if I uncommend
```
# See https://stackoverflow.com/a/44753457
# - name: Reset ansible connection after group changes
#   meta: reset_connection
#   when: ansible_version.major = 2
#     and ansible_version.minor >= 3
```

it runs

**Comment by dsander** [comment]
This seems to be an issue in ansible: https://github.com/ansible/ansible/issues/27520

**Comment by atosatto** [comment]
Give me the weekend to troubleshoot it. I am sure we can find a workaround
@Vad1mo what version of Ansible are you using?

**Comment by Vad1mo** [comment]
I am on the latest ansible 2.4.1.
Right now I just disabled the feature.

**Comment by pkelleratwork** [comment]
is there any new info on this issue? thx.

**Comment by atosatto** [comment]
I believe this has been fixed with `v2.0.0`.
In case you find this issue still affecting your setup, feel free to re-open it!
